maven-java/maven-archetype-akkasls-event-sourced-entity/src/main/resources/archetype-resources/src/main/proto/myentity_api.proto

```diff
@@ -7,26 +7,26 @@ import "google/api/annotations.proto";
 
 package $package;
 
-option java_outer_classname = "MyEntityApi";
+option java_outer_classname = "CounterApi";
 
 message SetValueCommand {
-    string entity_id = 1 [(akkaserverless.field).entity_key = true];
+    string counter_id = 1 [(akkaserverless.field).entity_key = true];
     int32 value = 2;
 }
 
 message GetValueCommand {
-    string entity_id = 1 [(akkaserverless.field).entity_key = true];
+    string counter_id = 1 [(akkaserverless.field).entity_key = true];
 }
 
-message MyResult {
+message CurrentCounter {
     int32 value = 1;
 }
 
-service MyServiceEntity {
+service CounterService {
     option (akkaserverless.service) = {
         type : SERVICE_TYPE_ENTITY
-        component : ".domain.MyEntity"
+        component : ".domain.Counter"
     };
     rpc SetValue(SetValueCommand) returns (google.protobuf.Empty);
-    rpc GetValue(GetValueCommand) returns (MyResult);
+    rpc GetValue(GetValueCommand) returns (CurrentCounter);
```

Comment:
also methods should follow the value entity variant

```proto
rpc Increase (IncreaseValue) returns (google.protobuf.Empty);
rpc Decrease (DecreaseValue) returns (google.protobuf.Empty);
rpc Reset (ResetValue) returns (google.protobuf.Empty);
rpc GetCurrentCounter (GetCounter) returns (CurrentCounter);
```

Reply:
done now